spec/function.dd

```diff
@@ -2054,14 +2054,12 @@ void func(T)(T x)
 {
     writeln(x);
 }
-
```

Comment:
I found where the problem lies an a workaround. Removing `\lstset{breaklines=true}` from latex.ddoc makes everything work nicely. However, that foregoes an important feature - long lines were nicely split.

Anyway, this is an ongoing annoyance that must be addresse. I'll explore more workarounds in a future PR.